src/post_processing/virtual_reservoir_revenue.jl

```diff
@@ -0,0 +1,664 @@
+# tenho que me preocupar com ex ante ex post?
+# preço zonal, nodal?
```

Comment:
Eu implementei as coisas de receitas antes de ter isso, vale a pena a gnt revisar sim, mas pode fazer em outro PR se for complicar mto